packages/components/src/components/data-table/table-row-info.tsx

```diff
@@ -14,21 +16,30 @@ type TTableRowIndex = {
 
 const TableRowInfo = ({ replace, is_footer, cells, className, is_dynamic_height, measure }: TTableRowIndex) => {
     const [show_details, setShowDetails] = React.useState(false);
+    const debouncedHideDetails = useDebounce(() => setShowDetails(false), 5000);
 
     const toggleDetails = () => {
         if (replace) {
             setShowDetails(!show_details);
+            debouncedHideDetails();
         }
     };
+
+    const toggleDetailsDecorator = (e?: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>) => {
```

Comment:
why do we create a function just to execute a shared function we already created ?

Reply:
its not a big deal. Just wondering